wagtail/admin/views/workflows.py

```diff
@@ -393,24 +399,86 @@ def delete_action(self):
         self.object.deactivate(user=self.request.user)
 
 
-def usage(request, pk):
-    workflow = get_object_or_404(Workflow, id=pk)
+class WorkflowPageFilterSet(PageIndexView.filterset_class):
+    content_type = PageTypeFilter()
```

Comment:
I opted to create a new filterset class rather than reusing the `ExplorablePageFilterSet` as-is, just in case there are new filters added to `ExplorablePageFilterSet` that are specific to the explorable listing.